feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/quicksettings/QuickSettingsEnums.kt

```diff
@@ -96,6 +95,12 @@ enum class CameraFlashMode : QuickSettingsEnum {
         override fun getImageVector() = Icons.Filled.FlashOn
         override fun getTextResId() = R.string.quick_settings_flash_on
         override fun getDescriptionResId() = R.string.quick_settings_flash_on_description
+    },
+    LLB {
```

Comment:
again something like `LOW_LIGHT` would help with readability instead of just `LLB`